docs/okeps/okep-5224-connecting-udns/okep-5224-connecting-udns.md

```diff
@@ -0,0 +1,1652 @@
+to `blue_blue.network_ovn-worker` and `04aa1e0b-bfec-4270-94f0-4b6d511b63a3`
+to `green_green.network_ovn-worker` for the blue pods to be able to
+talk to green service and vice versa. The switches will DNAT the traffic
+and the policies and routes we added in the **Pods** section will take
+care of routing the DNAT-ed traffic to the correct network.
+
+```yaml
+sh-5.2# ovn-nbctl ls-lb-add blue_blue.network_ovn-worker ab3397dd-6c4f-41bd-9239-60c73c25aaee
+sh-5.2# ovn-nbctl ls-lb-add green_green.network_ovn-worker 04aa1e0b-bfec-4270-94f0-4b6d511b63a3
+sh-5.2# ovn-nbctl ls-lb-list blue_blue.network_ovn-worker
+UUID                                    LB                  PROTO      VIP                 IPs
+04aa1e0b-bfec-4270-94f0-4b6d511b63a3    blue_blue.networ    tcp        10.96.210.158:80    103.103.0.5:8080,103.103.2.5:8080
+ab3397dd-6c4f-41bd-9239-60c73c25aaee    green_green.netw    tcp        10.96.69.71:80      104.104.1.4:8080,104.104.2.4:8080
+sh-5.2# ovn-nbctl ls-lb-list green_green.network_ovn-worker
+UUID                                    LB                  PROTO      VIP                 IPs
+04aa1e0b-bfec-4270-94f0-4b6d511b63a3    blue_blue.networ    tcp        10.96.210.158:80    103.103.0.5:8080,103.103.2.5:8080
+ab3397dd-6c4f-41bd-9239-60c73c25aaee    green_green.netw    tcp        10.96.69.71:80      104.104.1.4:8080,104.104.2.4:8080
+```
+
+Case3: `ClusterIPServiceNetwork` is set and `PodNetwork` is not set
+
+This means user is asking for partial connectivity between UDNs
+through serviceCIDRs. They don't want direct connectivity between
+all their applications.
+
+We will use ACLs in `Tier0` to allow traffic to service CIDR
+and drop all `connected_udn_subnet` pod traffic. These ACLs
+will be added on the local node switches for layer3 networks
+and the distributed flat switch for layer2 networks.
+They are applied in the `from-lport` direction for traffic
+originating from the pod's ports.
+
+```yaml
+_uuid               : e88434bf-70c2-4853-810e-ecb2a64aaaa7
+action              : pass
+direction           : from-lport
+external_ids        : {}
+label               : 0
+log                 : false
+match               : "ip4.dst==10.96.0.0/16"
+meter               : []
+name                : []
+options             : {}
+priority            : 5000
+sample_est          : []
+sample_new          : []
+severity            : []
+tier                : 0
+---
+sh-5.2# ovn-nbctl list address-set connected_udn_subnets
+_uuid               : d340f83e-0adb-4f8c-9196-1e36ddcda7e9
+addresses           : ["103.103.0.0/16", "104.104.0.0/16", "105.105.0.0/16"]
+external_ids        : {}
+name                : connected_udn_subnets
+---
+_uuid               : cfe7b1cd-25cc-4f4f-8d15-f7b978ef43a1
+action              : drop
+direction           : from-lport
+external_ids        : {}
+label               : 0
+log                 : false
+match               : "ip4.src==$connected_udn_subnets && ip4.dst==$connected_udn_subnets && ct.new"
```

Comment:
so did we establish it is required or just didn't finalize the investigation yet?